Common++/header/Logger.h

```diff
@@ -187,7 +193,11 @@ namespace pcpp
 		 * Set all PcapPlusPlus modules to a certain log level
 		 * @param[in] level The log level to set all modules to
 		 */
-		void setAllModulesToLogLevel(LogLevel level) { for (int i=1; i<NumOfLogModules; i++) m_LogModulesArray[i] = level; }
+		void setAllModulesToLogLevel(LogLevel level)
+		{
+			for (int i = 1; i < NumOfLogModules; i++)
+				m_LogModulesArray[i] = level;
```

Comment:
Is there a way to always enforce curly braces even if there's only one statement?